cdbot/cogs/help.py

```diff
@@ -1,11 +1,20 @@
-from discord import Colour, Embed
+from discord import Colour, Embed, utils
 from discord.ext import commands
 
 
 class EmbeddedHelpCommand(commands.HelpCommand):
     def __init__(self):
         super().__init__(command_attrs={'help': 'Gives detailed information about a command.'})
 
+    async def command_callback(self, ctx, *, command=None):
+        bot = ctx.bot
+        if command is not None:
```

Comment:
Could `if command is not None` be `if command` instead?

Reply:
yes and also no

Reply:
Truthiness go brrrrr